docker-compose.yml

```diff
@@ -1,25 +1,63 @@
-version: '2'
+version: '3.1'
 
 services:
+
     postgres:
-        image: postgres:9.5-alpine
+        image: postgres:11.6-alpine
+        container_name: postgres
+        hostname: postgres
+        restart: always
         environment:
-            POSTGRES_USER: zotonic
+           - POSTGRES_USER=zotonic
+           - POSTGRES_DB=zotonic
+           - POSTGRES_PASSWORD=zotonic
+           - PGDATA=/var/lib/postgresql/data/pgdata
         volumes:
-            - ./data:/data
-        working_dir: /data
+           - "./docker-data/postgresql:/var/lib/postgresql/data/pgdata"
 
     zotonic:
-        image: zotonic/zotonic-dev
-        user: zotonic
-        privileged: true
+        image: zotonic/zotonic-dev:latest
+        # user: ${CURRENT_UID}
```

Comment:
Removing this will run the container as root, which is considered bad practice. Doesn’t running as `zotonic` work?

Reply:
Yes, should run as the local user anyway.

Note that the `CURRENT_UID` doesn't seem to be available on macOS

Reply:
Better to move the user selection to the Dockerfile.

Reply:
In the container we can run as `zotonic`.
The container itself as the current user - shouldn't need any root permissions.

Reply:
If you don’t mount anything at `/etc/zotonic`, the default config that is built into the container image is used.

Reply:
I am not using any `USER` directive now and all files in the host OS are created using my local user. So I assume everything is ok now.